study.md

```diff
@@ -31,7 +31,8 @@ manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+array = [23,56,3,7]
+array.sort.reverse from try ruby
```

Comment:
Since we want to modify the original array, the invocations of `sort` and `reverse` should be `sort!` and `reverse!`, respectively.